funcx_endpoint/tests/smoke_tests/test_s3_indirect.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from funcx_endpoint.executors.high_throughput.funcx_worker import MaxResultSizeExceeded
+
+
+def large_result_producer(size: int) -> str:
+    return bytearray(size)
+
+
+def large_arg_consumer(data: str) -> int:
+    return len(data)
+
+
+test_cases = [200, 2000, 20000, 200000]
```

Comment:
Alternatively, you could just inline the list in the parametrize usage. That's what I'd tend to do -- the tests using this list are testing different things. It would need to be rewritten if/when the behaviors of those parts of the service diverge anyway.